pkg/planner/core/operator/logicalop/logical_index_scan.go

```diff
@@ -42,6 +42,7 @@ type LogicalIndexScan struct {
 
 	Index          *model.IndexInfo
 	Columns        []*model.ColumnInfo
+	ColIdxsByName  map[string]int
```

Comment:
i saw the adding of ColIdxsByName into the PhysicalTableScan and LogicalIndexScan, but any usage of them?